src/main/java/com/code_intelligence/jazzer/mutation/mutator/collection/MapMutatorFactory.java

```diff
@@ -126,32 +130,101 @@ public void initInPlace(Map<K, V> reference, PseudoRandom prng) {
       }
     }
 
+    private void eraseRandomChunk(Map<K, V> map, PseudoRandom prng, int mapSize) {
+      int upperLimit = Math.max(1, Math.min(mapSize - minSize, mapSize / 2));
+      int chunkSize = upperLimit == 1 ? 1 : prng.closedRange(1, upperLimit);
+      int chunkOffset = prng.closedRange(0, mapSize - chunkSize);
+      List<K> tmpRemStore = new ArrayList<>(chunkSize);
+      map.entrySet().stream().skip(chunkOffset).limit(chunkSize).forEach(entry -> {
+        tmpRemStore.add(entry.getKey());
```

Comment:
You can use `.collect(Collectors.toCollection(new ArrayList<>(chunkSize)))` instead of creating the list and then using `forEach`.